### ISSUE TITLE
No Screenshots

### ISSUE THREAD

**Issue by Lord-Fredrick** [budgie-desktop/budgie-site#12]
Website doesn't have screenshots easily available which is kinda useful if you are looking at DE's.

**Comment by KadenGordon** [comment]
Yeah... I'd rather not just install it without first knowing what it looks like

**Comment by JoshStrobl** [comment]
You don't need to install it, just grab the Solus ISO.

**Comment by ghost** [comment]
Same thing, downloading a 2 GB ISO image just to see how Budgie looks it's a bit too much. The Budgie repo already has screenshots, so why not use those?
